numpy/ma/core.py

```diff
@@ -2870,6 +2870,11 @@ def __new__(cls, data=None, mask=nomask, dtype=None, copy=False,
         else:
             # Case 2. : With a mask in input.
             # If mask is boolean, create an array of True or False
+            
+            # If mask is None, cast it to False
+                    if mask is None:
```

Comment:
Linter is unhappy, make sure to use the right indentation.

Reply:
Oopsie.
Must be good now!